backend/kuberay/README.md

```diff
@@ -32,6 +32,70 @@ the
     avoid S3-like filesystems as that is known to cause failures
     in **transformers**. Use a NFS or GPFS-backed PersistentVolumeClaim instead.
 
+### Configuring a Kubernetes ServiceAccount for the RayCluster
+
+The default Kubernetes ServiceAccount created for a RayCluster does not
+have enough permissions for an ado actuator to create Kubernetes resources
+(e.g., deployments, pods, services, etc.) as part of its operations.
+Users are required to create a custom ServiceAccount bound to a Role with
+enough permissions, before creating the RayCluster, to avoid runtime errors.
+Below is an example Service account bound to a Role that allows
+the monitoring, creation and deletion of pods, deployments and services.
+It also provides access to the raycluster resources.
+
+<!-- markdownlint-disable-next-line code-block-style -->
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ray-deployer
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: ray-deployer
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: ray-deployer
+subjects:
+  - kind: ServiceAccount
+    name: ray-deployer
+namespace: cp-testing
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: ray-deployer
+  namespace: cp-testing
+rules:
+  - apiGroups: ["ray.io"]
+    resources:
+      - rayclusters
+    verbs: ["get", "patch"]
+  - apiGroups: ["apps"]
+    resources:
+      - pods
+      - deployments
+    verbs: ["get", "create", "delete", "list", "watch", "update"]
+  - apiGroups: [""]
+    resources:
+      - services
+    verbs: ["get", "create", "delete", "list", "watch", "update"]
+```
+
+From root of the ado project run the below command:
+
+    kubectl apply -f backend/kyberay/service-account.yaml
+
+This will create a Service account named `ray-deployer`.
+This name is required later while
+[deploying the RayCluster](#example-kubernetes-cluster-with-4-nodes-8-gpus-each).
```

Comment:
```suggestion
This will create a ServiceAccount named `ray-deployer`. We will reference this
name when
[deploying the RayCluster](#example-kubernetes-cluster-with-4-nodes-8-gpus-each).
```